main.go

```diff
@@ -75,17 +86,19 @@ func main() {
 	if err != nil {
 		log.Fatalln("can't create bucket: ", err)
 	}
+
 	app := cli.NewApp()
 	app.Name = "gw-manager"
 	app.Version = "1.0.0"
-	app.Usage = "Sample app shows how to manage Gateway"
+	app.Usage = "Sample app shows how to manage Gateway. Specify the path of config file with env variable GWM_CONFIG" +
```

Comment:
People who are looking for the way of change the config file path might not find it.

Presumably, README is the better place to put the description.